### DISCUSSION TITLE
Make validation compatible with the Constraint Validation API

### DISCUSSION THREAD

**Discussion by MarkMcDonald** [umbraco/Umbraco.Forms.Issues#1512] [Ideas]
Currently, HTMLFormElement.checkValidity() will report false even though the JQuery validate methods validate the form in cases where there are required conditional fields.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement

Constraint validation still validates fields even when hidden, but should not if disabled so disabling the field as well as hiding may well resolve the issue.

**Comment by TvGessel** [comment]
Could you provide some specific examples of how you'd like to see this working? It would help us better understand the requirement.
